tests/console/consoletest_finish.pm

```diff
@@ -58,7 +58,12 @@ sub run() {
         }
         wait_idle;
         mouse_hide(1);
-        assert_screen 'generic-desktop';
+        if (check_var("DESKTOP", "awesome")) {
+            assert_screen 'displaymanager', 200;
```

Comment:
hmm...interesting part, this test I assumed it will brings x11_login test, so in this stage it must stop here and skip logging in but log in to awesome in next step ie. x11_login. I didn't say it's strange or wrong, it's just a rare case use x11_login :)